readme/BREAKING_CHANGES.markdown

```diff
@@ -1470,4 +1470,26 @@ This removal of the configuration has no replacement.
 
 ### Why was this change made?
 
-Liferay decided to not support this configuration.
\ No newline at end of file
+Liferay decided to not support this configuration.
```

Comment:
@tinatian Do we still need to record this breaking change if the interface `SoapDescriptorBuilder` is a provider type?